client/view/Form/Form.js

```diff
@@ -47,30 +45,20 @@ function handleFormSubmit(e) {
   changeUrl(query, region)
 
   e.preventDefault()
```

Comment:
oops, after some PR we have 2 `e.preventDefault` in `handleFormSubmit`

Reply:
Fixed e8a926b